experimental/PropertyDDS/packages/property-inspector-table/webpack.prod.js

```diff
@@ -2,194 +2,8 @@
- * @property {string} [tsConfigFile] The name of the TypeScript configuration file (tsconfig). Either as an absolute
- *  path or relative to the project root. Can also be set on the command line as --tsConfigFile=[tsConfigFile].
- * @property {object} [merge={}] An object that contains valid webpack configuration options. Will be merged with the
- *  default options defined in this config.
- */
-
-/**
- * The webpack base configuration for Application Framework libraries (e.g. AppComponents).
- * @param {AppfwLibraryParameters} args Configuration parameters.
- * @return {array<object>} An array of webpack configurations with two entries. The first one is for a non-minified
- *  development build, the second one is a minified production bundle.
- */
-const CommonWebpackLibTSConfig = function (args) {
-  args = _.extend({ fileType: 'ts' }, baseConfig.commonParams, { distTypesPath: 'dist/types' }, args);
-  const params = getParams(args);
-
-  const commonLibConfig = {
-    devtool: 'source-map',
-    output: {
-      library: params.name,
-      path: params.distPath
-    },
-    externals: {
-      "@fluid-experimental/property-properties": {
-        amd: "@fluid-experimental/property-properties",
-        commonjs: "@fluid-experimental/property-properties",
-        commonjs2: "@fluid-experimental/property-properties",
-        root: ["Properties"]
-      }
-    },
-    module: {
-      rules: [
-        {
-          test: /\.[tj]sx?$/,
-          exclude: [/node_modules/, /\.min\.js$/],
-          use: {
-            loader: require.resolve('babel-loader'),
-            options: {
-              babelrc: false,
-              presets: [
-                [
-                  '@babel/preset-env',
-                  { targets: { 'ie': 11 }, modules: 'commonjs' }
-                ]
-              ],
-              plugins: ['@babel/plugin-transform-runtime']
-            }
-          }
-        }
-      ]
-    }
-  };
-
-  const tsLibConfig = baseConfig.TSConfig(params.fileType, params.generateDeclarations, params.distTypesPath,
-    params.tsConfigFile);
-
-  const libConfig = merge(baseConfig.BaseConfig, commonLibConfig, tsLibConfig, params.merge);
-
-  const isFile = fs.statSync(params.dir).isFile();
-  const entryFile = path.resolve(params.dir, isFile ? '' : './webpack.index.' + params.fileType);
-  let devEntry = {};
-  devEntry[params.name] = entryFile;
-  let prodEntry = {};
-  prodEntry[params.name + '.min'] = entryFile;
-
-  const pathsToClean = [params.distPath];
-  if (params.fileType === 'ts' && params.generateDeclarations) pathsToClean.push(params.distTypesPath);
-
-  return [
-    merge(libConfig, {
-      mode: 'development',
-      entry: devEntry,
-      plugins: [
-        new CleanWebpackPlugin({ cleanOnceBeforeBuildPatterns: pathsToClean }, process.cwd())
-      ]
-    }),
-    merge(libConfig, {
-      mode: 'production',
-      entry: prodEntry
-    })
-  ];
+module.exports = {
```

Comment:
While fluid has been using the pattern of having these extra config files with just this little data in them for prod vs dev, we haven't really gotten much value out of it, so feel free to just inline this content into the larger webpack files if you want. I think that would help reduce confusion since you have a lot of webpack config related files here.